NugetLightRequest.cs

```diff
@@ -1587,7 +1612,13 @@ into newGroup
                     }
                     else
                     {
-                        pkgs = from p in pkgs where p.IsLatestVersion select p;
+                        SemanticVersion latest = new SemanticVersion("0.0.0");
+                        foreach (var p in pkgs) {
+                            if (p.Version > latest) {
+                                latest = p.Version;
+                            }
+                        }
+                        pkgs = from p in pkgs where p.Version == latest select p;
```

Comment:
Also this seems to be doing more work than it really needs to. You scan the whole of pkgs once looking for the largest version, then you scan it again to find all the packages which match that version. Is it possible for multiple items in the list to have the same Version? If not you could just remember the package you found rather than refiltering the list

Reply:
It’s not possible to have multiple versions.... saving the package instead of the version does make more sense.